bundles/AdminBundle/Controller/Admin/DataObject/ClassController.php

```diff
@@ -1232,6 +1237,7 @@ public function objectbrickTreeAction(Request $request)
             } else {
                 if ($forObjectEditor) {
                     $layout = $item->getLayoutDefinitions();
+                    DataObject\Service::enrichLayoutDefinition($layout, $object);
```

Comment:
Is it really required? as enrichment is done later in the block https://github.com/pimcore/pimcore/blob/6d1e4d376f70e263636d25a47edb0fe6853b1051/bundles/AdminBundle/Controller/Admin/DataObject/ClassController.php#L1256

Reply:
good point